settings.gradle.kts

```
@@ -1,5 +1,11 @@
 rootProject.name = "probation-integration-services"
-include("libs:probation-integration-commons", "prison-case-notes-to-probation", "workforce-allocations-to-delius")
+include(
+    "libs:probation-integration-commons",
+    "libs:probation-integration-dev-tools",
```

Minor thing - do we need the `probation-integration` prefix for these?
```suggestion
    "libs:commons",
    "libs:dev-tools",
```